database/migrations/2022_05_06_190816_create_permission_tables.php

```diff
@@ -0,0 +1,140 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Spatie\Permission\PermissionRegistrar;
+
+class CreatePermissionTables extends Migration
```

Comment:
Please be consistent with other migration files and convert it to anonymous class.